### PR TITLE
Add @storybook/addon-viewport

### PR DESCRIPTION
**Pull request by damassi**
Adds Storybook's official responsive plugin which allows one to see stories at different dimensions, and control the dimension programmatically:

```js
storiesOf("Components/NavBar", module).add("Mobile NavBar", () => {
  return <NavBar />
}, {
  viewport: { 
    defaultViewport: 'iphonex' 
  }
})
```

Also bumps things to latest. 

viewport

56939176-c9e27180-6abb-11e9-8e7f-7708e22d7f42

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+addDecorator(wrapWithThemeProvider)
 
-addDecorator(StoryRouter());
+addDecorator(StoryRouter())
 
 setOptions({
   name: 'Are.na',
   url: 'https://www.are.na',
-  sortStoriesByKind: true,
+  showAddonPanel: false,
```
Comment (damassi) [inline .storybook/config.js]: This just collapses the annoying empty `Actions` panel at the bottom of the screen.

Comment (dzucconi) [reply .storybook/config.js]: It still toggleable tho? (I do use it for certain components.) But I did notice some super annoying buggy behavior where it remembers position after you close your dev console.

**Comment by dzucconi** [comment]
👍 👍

File: .storybook/config.js (hunk 1 above)
Comment (damassi) [reply .storybook/config.js]: Yup, everything is contained inside of the nav (and the hotkey `A`): 

Screen Shot 2019-05-01 at 1 56 56 PM

(

Comment (dzucconi) [reply .storybook/config.js]: Ooo lovely

**Pull request merged**